src/engine/map/MapFunctions.hxx

```diff
@@ -4,6 +4,7 @@
 // #include "WindowManager.hxx"
 // #include "basics/Point.hxx"
 #include "../Map.hxx"
+#include "audio/AudioConfig.hxx"
```

Comment:
It looks like all you really did in this file is add a `levelHeight` function header--so I don't really see why it needs `AudioConfig`?

Reply:
Yeah, this one was probably me pressing enter too quickly and my IDE auto-importing AudioConfig, I'll patch it out.

Reply:
I found out why AudioConfig was included. It seems like this was the shortest path that my LSP could find that included `<Vector>`, which I used in the `levelTerrain` function. It's now properly included.